src/storage-client/src/controller.rs

```diff
@@ -3168,44 +3170,132 @@ where
         Ok(())
     }
 
-    /// On each element of `collections`, install a read hold on all of the `storage_dependencies`.
-    fn install_dependency_read_holds<I: Iterator<Item = GlobalId>>(
+    /// On each element of `collections`, install a read hold on all of the
```

Comment:
🙌 always like seeing more prose documentation like this!

Reply:
ty